tests/lib/rules/no-invalid-template-root.js

```diff
@@ -90,23 +102,23 @@ tester.run('no-invalid-template-root', rule, {
     },
     {
       filename: 'test.vue',
-      code: '<template><div v-if="foo"></div></template>',
-      errors: ["The template root requires the next element which has 'v-else' directives if it has 'v-if' directives."]
+      code: '<template><slot></slot></template>',
+      errors: ["The template root disallows '<slot>' elements."]
     },
     {
       filename: 'test.vue',
-      code: '<template><div v-if="foo"></div><div v-else-if="bar"></div></template>',
-      errors: ["The template root requires the next element which has 'v-else' directives if it has 'v-if' directives."]
+      code: '<template><template></template></template>',
+      errors: ["The template root disallows '<template>' elements."]
     },
     {
       filename: 'test.vue',
-      code: '<template><slot></slot></template>',
-      errors: ["The template root disallows '<slot>' elements."]
+      code: '<template src="foo.html">abc</template>',
+      errors: ["The template root which has 'src' attribute requires empty."]
```

Comment:
What do you think about amending this message to be: `The template root with 'src' attribute is required to be empty.`